### ISSUE TITLE
Misunification of A00851-003 (T6-2168) and A02977-001 (GKX-0249.04) (U+215D3)

### ISSUE THREAD

**Issue by hfhchan** [hfhchan/irg#180]
T6-2168 is a mis-unified form of two characters.  Based on current CNS11643, the pronunciations are given as tài (variant of 太) and lì (variant of 立) 

Glyph in code charts:
![image](https://user-images.githubusercontent.com/8191296/80909827-f9c2f580-8d5d-11ea-94f7-eaffc96b33af.png)

The source GKX-0249.04 is the variant of "立".

In the MOE dictionary, the form ⿱大一 is given as variant of 立 (A02977-001) while the form ⿵大一 is given as variant of 太 (A00851-003).  A00851-003 is currently using U+215D3

Therefore, there are two suggested treatment methods: 

1. Keep the shape ⿵大一 at T6-2168, preserve the pronunciation of tài;
    Separately code ⿱大一 with pronunciation lì;
    Dis-unify T6-2168 from U+215D3, and re-encode as a new character or IVS to 太.

or

2. Change the shape of T6-2168 to ⿱大一 and keep the pronunciation of lì only;
    Separately code ⿵大一 with pronunciation tài;
    Keep T6-2168 at U+215D3.
    Re-encode the new character or IVS to 太.

and modify the MOE dictionary to use U+215D3 in A02977-001, and PUA / new char for  A00851-003.

**Comment by wtn** [comment]
Option 2!

**Comment by hfhchan** [comment]
The shape is ⿱大一 encoded at 12-223B in CNS11643 with pronunciation lì:
![image](https://user-images.githubusercontent.com/8191296/81193168-a1a22280-8fed-11ea-96bc-0e87a8d7dd2e.png)
https://www.cns11643.gov.tw/wordView.jsp?ID=795195

So Option (1) would be preferred.
- Keep the shape ⿵大一 at T6-2168, preserve the pronunciation of tài;
- Dis-unify T6-2168 from U+215D3, and re-encode it as a new character or IVD
- Unify TB-223B to U+215D3.

**Comment by wtn** [comment]
Right. Sorry, I misunderstood separating in T-source as separating in UCS.
